404.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Docker docs</title>
+</head>
+<body>
+    <p>We have moved away from the <strong>docker.github.io</strong> domain. Please visit <a href="https://docs.docker.com">docs.docker.com</a> instead.</p>
+
+    <script>
+        window.location.replace("https://docs.docker.com" + window.location.pathname);
```

Comment:
Not sure we want to remove the original URL from history; perhaps use `.assign()` instead?

```javascript
window.location.assign("https://docs.docker.com" + window.location.pathname);
```